### PR TITLE
Added SELECT2_JSON_ENCODER option in Django settings.py

### PR DESCRIPTION
**Pull request by N1K1TAS95**
Added support to set a custom JSON encoder via Django settings.py

### CODE REVIEW

File: django_select2/conf.py [hunk 1]
```diff
@@ -198,6 +198,32 @@ class Select2Conf(AppConf):
...
+        from django.core.serializers.json import DjangoJSONEncoder
+        from hashid_field import Hashid
+        
+        class HashidJSONEncoder(DjangoJSONEncoder):
+            def default(self, o):
+                if isinstance(o, Hashid):
+                    return str(o)
+                return super().default(o)
+                
+    And set SELECT2_JSON_ENCODER to 'app_name.econders.HashidJSONEncoder' in settings.py of your project.
```
Comment (codingjoe) [inline django_select2/conf.py:223]:
```suggestion
    :class:`.JSONEncoder` used to generate the API response for the model widgets.
    
    A custom JSON encoder might be useful when your models uses
    a special pirmary key, that isn't serializable by the default encoder.
```

**Review by codingjoe — COMMENTED** [review]
Excellent! This should do the trick. I left you a suggestion for your documentation. I'd like to keep a little shorter, while adding more backlinks to the Python docs. They actually have a pretty good example on how to extend encoders. We use inter-sphinx to create those links, if classes are marked accordingly in the docs.

**Comment by codingjoe** [comment]
Oh an @N1K1TAS95 you'll need to move the docstring under the attribute, to make it work. I'd recommend running `python setup.py build_sphinx` to check the docs locally.

**Comment by N1K1TAS95** [comment]
@codingjoe I've moved attribute docs under the attribute itself. 
`python setup.py build_sphinx` compiles and docs are updated with new attribute.

**Review by codingjoe — APPROVED** [review]
Woop woop :tada: let's ship it

**Comment by N1K1TAS95** [comment]
> Woop woop 🎉 let's ship it

Thanks!

**Pull request merged**